app/kontres/models/bookable_item.py

```diff
@@ -0,0 +1,18 @@
+import uuid
+
+from django.db import models
+
+from app.common.enums import AdminGroup, Groups
+from app.common.permissions import BasePermissionModel
+from app.util.models import BaseModel
+
+
+class BookableItem(BaseModel, BasePermissionModel):
+    write_access = AdminGroup.admin()
+    read_access = [Groups.TIHLDE]
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    name = models.CharField(max_length=20)
+    description = models.TextField(blank=True)
+
+    def __str__(self):
+        return self.name
```

Comment:
Legg til classmethod med has_permission for update, destroy, retrieve, read og update